configs/ViT-B-16.py

```diff
@@ -38,7 +38,7 @@ def get_config():
     config.image_key = 'image'
 
     # dataloader
-    config.num_workers = 48
+    config.num_workers = 8
```

Comment:
why the reduction? are you seeing contention with the cores?

Reply:
Yeah I kept getting some weird bus error, and this fixed it (but not sure 8 is the best value, just what worked)